compiler/testData/diagnostics/tests/EnumEntryAsType.fir.kt

```diff
@@ -6,49 +6,49 @@ enum class Color {
     }
 }
 
-class MyColor(val x: <!UNRESOLVED_REFERENCE!>Color.RED<!>, y: <!UNRESOLVED_REFERENCE!>Color.RED<!>) : <!UNRESOLVED_REFERENCE!>Color.RED<!> {
+class MyColor(val x: <!ENUM_ENTRY_AS_TYPE!>Color.RED<!>, y: <!ENUM_ENTRY_AS_TYPE!>Color.RED<!>) : <!ENUM_ENTRY_AS_TYPE!>Color.RED<!> {
```

Comment:
https://punzki.github.io/Kotlin-DiagnosticTestsDiff/delta-IS_ENUM_ENTRY/ENUM_ENTRY_AS_TYPE.html tells me that in FE 1.0, the diagnostic was reported on the enum entry name only (`RED`), not the fully-qualified name (`Color.RED`). I think you can just use `SELECTOR_BY_QUALIFIED` positioning strategy.

Reply:
Thanks! I have pushed another commit to fix this.